index.html

```diff
@@ -156,7 +156,26 @@ <h3 class="frameworks"><span>Frameworks</span><img src="./images/arrow-right.svg
                   <h3 class="skills"><span>Skills</span><img src="./images/arrow-right.svg" alt=""></h3>
                 </div>
               </section>
-            
-  
+
+              <!--Contact info section-->
+              <section class="contact-info">
+                <article class="contact-headings"> 
+                  <h2>Contact me</h2>
+                   <p> If you have an application you are interested in developing, 
+                     a feature that you need built or a project that needs coding.
+                      I’d love to help with it  </p> 
+                  </article>
+
+                <form action="https://formspree.io/f/xgeddrdw" method="POST">
+
+                <input type="text" name="username"  maxlength="30" placeholder="Yeremias" required>
+
+                <input type="email" name="email" placeholder="natajayanj14@gmail.com" required>
+
+                <textarea name="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
```

Comment:
- [ ] it would be better if you use the right color for the text like in the Figma design 
![image](https://user-images.githubusercontent.com/61790587/171231963-abe85228-22fa-4e6e-839a-d8b22be07ce2.png)